steps/android-instrumented-test/assets/icon.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```

Comment:
The icon seems to have a kind of 'margin' on the top and bottom, which seems odd. Compare it to [android-build ](https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/android-build/assets/icon.svg?short_path=accb108 ) for example.
There also seem to be embedded png files in the svg is that acceptable? Technically still an svg as [required](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#adding-a-step-icon) per , but not vector graphics.

Reply:
That's what was provided to me from the design team, but I can ask them to make sure it's correct.